app/Models/Litter/Categories/Brand.php

```diff
@@ -120,15 +17,13 @@ public function types ()
             'applegreen',
             'asahi',
             'avoca',
-
```

Comment:
Keep the spaces between the first letter please. This was included for a reason- to make adding new ones easier